SafeExamBrowser.Browser.Contracts/IBrowserApplication.cs

```diff
@@ -30,5 +32,15 @@ public interface IBrowserApplication : IApplication
 		/// Event fired when the browser application detects a request to terminate SEB.
 		/// </summary>
 		event TerminationRequestedEventHandler TerminationRequested;
+
+		/// <summary>
+		/// Event fired when the user tries to focus the taskbar.
+		/// </summary>
+		event LoseFocusRequestedEventHandler LoseFocusRequested;
+
+		/// <summary>
+		/// Transfers the focus to the browser window.
+		/// </summary>
+		void Focus(bool forward);
```

Comment:
If `forward` is true, the first focusable element in the browser window receives focus (passing forward of focus). Otherwise, the last element.

I just added this to the docs comments.

Reply:
Perfect, thanks